### PR TITLE
Fix issue null log on saving Services and categories.

### PR DESCRIPTION
**Pull request by RomuloLlabresJr**
@ismaeljhon Added code to prevent null on slug, upon saving categories and services. Please kindly review.Thanks.

### CODE REVIEW

File: components/ServiceForm.vue [hunk 1]
```diff
@@ -13,7 +13,7 @@
                   name="Name"
                   :rules="'required'"
                 >
-                  <v-text-field v-model="form.name" :error-messages="errors">
+                  <v-text-field v-model="form.name" :error-messages="errors"  v-on:keyup="slugifyName" v-on:change="slugifyName">
```
Comment (ismaeljhon) [inline components/ServiceForm.vue]: You can use @keyup and @change here.. shorthand @RomuloLlabresJr

File: components/CategoryForm.vue [hunk 2]
```diff
@@ -32,7 +32,7 @@
             name="Name"
             :rules="'required'"
           >
-            <v-text-field v-model="form.name" :error-messages="errors">
+            <v-text-field v-model="form.name" :error-messages="errors" v-on:keyup="slugifyName" v-on:change="slugifyName">>
```
Comment (ismaeljhon) [inline components/CategoryForm.vue]: You can use @keyup and @change here.. shorthand @RomuloLlabresJr

**Review by ismaeljhon — CHANGES_REQUESTED** [review]

File: components/ServiceForm.vue (hunk 1 above)
Comment (RomuloLlabresJr) [reply components/ServiceForm.vue]: @ismaeljhon i checked update code to use shorthand on keyup

**Comment by RomuloLlabresJr** [comment]
@ismaeljhon I have updated the code. Please kindly review it thanks

**Review by ismaeljhon — APPROVED** [review]

**Pull request merged**